packages/common/src/components/TextboxClipboardWrapper/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import styled from 'styled-components';
+import Clipboard from 'clipboard';
+import { TextField, ITextField } from 'office-ui-fabric-react/lib/TextField';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { invokeGlobalErrorHandler } from '../../utilities/splash.screen';
+import { generateCryptoSafeRandom } from '../../utilities/misc';
+
+interface IProps {
+  text: string;
+  style?: React.CSSProperties;
+}
+
+interface IState {}
+
+class TextboxClipboardWrapper extends React.Component<IProps, IState> {
+  private clipboardButtonClassName: string;
+  private clipboard: Clipboard;
+
+  constructor(props: IProps) {
+    super(props);
+
+    this.clipboardButtonClassName = 'export-to-clipboard-' + generateCryptoSafeRandom();
```

Comment:
This way if you have multiple of this component on the same page, they won't conflict.
It doesn't technically need to be a crypoSafeRandom, it could be a counter as well.  But I didn't know where to put it.  Unless I wanted to create a clipboard wrapper on top of clipboard.js, and then I could have a static that I keep incrementing.

I'm open to ideas.  It does seem a bit weird/quirky the way I have it now, and I also wonder about the perf or creating a random number vs. just incrementing a counter (though we're probably talking about a millisecond or so, if that)

Reply:
Hmmm. I think an even better idea would be to say this is an excellent case of not your problem, and defer this problem to the person who would be creating your TextboxClipboardWrapper having them declare what's unique about it. 

To do this, I'd add a key prop to this component and then use that key in the generation of the clipboard/className